CMakeLists.txt

```diff
@@ -532,7 +532,7 @@ if(NOT SDL_FOREGROUNDING_SIGNAL STREQUAL "OFF")
 endif()
 
 # Compiler option evaluation
-if(USE_GCC OR USE_CLANG)
+if(USE_GCC OR USE_CLANG OR QNX)
```

Comment:
What compiler does qnx use and what is its [CMake compiler id](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_COMPILER_ID.html)?

Reply:
It's `QCC`. I made a change to detect that and define `USE_QCC` for consistency.